devicetypes/samsungsds/samsung-smart-doorlock.src/i18n/messages.properties

```diff
@@ -13,8 +13,8 @@
 #  under the License.
 
 # Chinese
-'''Samsung Smart Doorlock'''.zh-cn=SDS联网型智能锁
+'''Samsung Door Lock'''.zh-cn=SDS联网型智能锁
 
 # Korean (ko)
-'''Samsung Smart Doorlock'''.ko=삼성 스마트 도어락
+'''Samsung Door Lock'''.ko=삼성 스마트 도어락
```

Comment:
my reservation for changing join names that are translated is that if a user has left their device with the default install name, it will immediately stop being translated once this change is deployed

Reply:
Good point. We should leave the existing translation and add new ones for the new join names.